subscriptions/views.py

```diff
@@ -1,26 +1,40 @@
-from rest_framework import viewsets, status
+from rest_framework import filters, viewsets, status
 from rest_framework.permissions import IsAuthenticated, IsAdminUser
 from rest_framework.views import APIView
 from rest_framework.response import Response
 from rest_framework.authtoken.models import Token
 from django.core.exceptions import ObjectDoesNotExist
 from django.contrib.auth.models import User
+import django_filters
 
 from .models import Subscription
 from .serializers import SubscriptionSerializer, CreateUserSerializer
 from .tasks import send_next_message, scheduled_metrics
 from seed_stage_based_messaging.utils import get_available_metrics
 
 
+class SubscriptionFilter(filters.FilterSet):
+    created_from = django_filters.IsoDateTimeFilter(
```

Comment:
This is what the identity store uses, so I assumed that that was the standard for the seed stack, and wanted to keep it the same as everywhere else. I'm not actually sure if this is really the case.